bot/persistence/database_connector.py

```diff
@@ -0,0 +1,89 @@
+"""Contains logic for connecting to and manipulating the database"""
+
+from sqlite3 import Error
+from typing import List, Union
+from bot import constants
+from .database_manager import DatabaseManager
+
+
+class DatabaseConnector():
+    """Class used to communicate with the database.
+     The database is created and initialized with, using the __init__ method
+     The other methods support getting or adding properties to the database.
+    """
+    _db_file: Union[str, None] = None
+
+    def __init__(self, db_file: str, init_script=None):
+        """ create a database connection to a SQLite database and create the default tables form the sql script in
+        init_db.sql
+
+        Args:
+            self: this object
+            db_file: The file name of the sqllite database file
```

Comment:
Proper names and abbreviations always in upper case e.g. `SQLite` or `SQL`. Dot is missing at the end.